src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerCredentialsTest.java

```diff
@@ -57,7 +57,8 @@ public void configRoundTripEmpty() throws Exception {
                 Collections.<DomainSpecification>singletonList(new DockerServerDomainSpecification()));
         DockerServerCredentials credentials = new DockerServerCredentials(CredentialsScope.GLOBAL, "foo", "desc", "", "", "");
         store.addDomain(domain, credentials);
-        j.submit(j.createWebClient().goTo("credential-store/domain/" + domain.getName() + "/credential/"+credentials.getId()+"/update")
+
+        j.submit(j.createWebClient().goTo("credentials/store/system/domain/" + domain.getName() + "/credential/"+credentials.getId()+"/update")
```

Comment:
I am surprised you have to edit the links here. Have the original links been deprecated in Credentials2.x ?